api/prd/caseFlag/models/caseFlagReference.model.ts

```diff
@@ -6,4 +6,6 @@ export interface CaseFlagReferenceModel {
   isParent: boolean;
   Path: string[];
   childFlags: CaseFlagReferenceModel[];
+  defaultStatus: string;
+  externallyActive: boolean;
```

Comment:
@connorpgpmcelroy - property name is not quite correct... should be `externallyAvailable` (see earlier PR review comment)